### DISCUSSION TITLE
/auth/callback/google timeout

### DISCUSSION THREAD

**Discussion by testvinder** [nextauthjs/next-auth#9546] [Help]
In production logs on Vercel, this error comes intermittently (several times a day). I'm not able to reproduce it. Login and signup with Google seems to work fine. I'm just wondering if anyone else has faced something similar or has any idea why it happens. I'm using the Sveltekit package.

> 
> [GET] /auth/callback/google?scope=email%20profile%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fuserinfo.email%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fuserinfo.profile%20openid&authuser=0&prompt=none&code=4%2F0AfJohXkoGW2zjj_iue0blgCSbZNXYo20acGDjEZRSOH4EphimLNQM3BD_7ChVUpusas_og status=504
> 
> Task timed out after 15.00 seconds

**Comment by testvinder** [comment]
Has anyone else faced this?

**Reply by fahrikurniawan99** [reply]
has this case been resolved ?
